sections/sprint-2/deploymentdiagram.tex

```diff
@@ -3,7 +3,7 @@ \section{Deployment Diagram}\label{sec:sprint2-deploymentdia}
 In order to further elaborate on the different components, a deployment diagram is constructed.
 \begin{figure}[H]
     \centering
-    \includegraphics[width=0.6\linewidth]{Deploymentdiagram.png}
+    \includegraphics[width=0.6\linewidth]{deploymentdiagram1.png}
```

Comment:
Hvis du har slettet det gamle deployment diagram. Kan det her så ikke bare hedde deploymentdiagram istedet for deploymentdiagram1